mne_bids/write.py

```diff
@@ -186,12 +190,15 @@ def _electrodes_tsv(raw, fname, kind, overwrite=False, verbose=True):
                             ('z', z),
                             ('size', sizes),
                             ])
-    else:
+    elif kind == 'eeg':
         data = OrderedDict([('name', names),
                             ('x', x),
                             ('y', y),
                             ('z', z),
                             ])
+    else:  # noqa
```

Comment:
Oh yeah oops, put it to `pragma: no cover` since idk if its necessary to test... just there to make sure an informative message gets passed back if a 'kind' is incorrectly passed.